### PR TITLE
feat: Dockerfile

### PR DESCRIPTION
**Pull request by kekrs**
Hey! Nice project you got there :)

I'm running an MC instance in Kubernetes and wanted to scratch an itch: Bluemap is unstable af, also generates enormous files. This project is the most perfect thing. <sub>With the small exception of nether/end support.</sub>

Usage:

```
docker build -f Dockerfile -t ghcr.io/neocturne/minedmap:latest
docker build -f Dockerfile.viewer -t ghcr.io/neocturne/minedmap-viewer:latest
docker run --rm -v ./data/world:/input -v ./output:/output ghcr.io/neocturne/minedmap:latest --sign-prefix '' /input /output
docker run -it --rm -v ./output/:/usr/share/nginx/html/data -p 8080:80 ghcr.io/neocturne/minedmap-viewer:latest
```
Finally open up localhost:8080 in a browser of your choice.

### CODE REVIEW

**Comment by neocturne** [comment]
Maybe also add the build/run instructions as comments in the Dockerfiles?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM docker.io/library/rust:alpine AS BUILDER
+
+WORKDIR /src
```
[the remark this answers is not in the document]
Comment (kekrs) [reply Dockerfile]: Agreed, done.

**Comment by kekrs** [comment]
> Maybe also add the build/run instructions as comments in the Dockerfiles?

Mount path of the viewer not intuitive, included a small hint. Other than that it should be straightforward for anyone using docker. These lines could be included in some kind of documentation, that would be a lot better place for this info.

**Comment by neocturne** [comment]
Thanks!

**Pull request merged**